code.html

```diff
@@ -5,6 +5,7 @@
     <link rel="stylesheet" type="text/css" href="style.css">
 </head>
 <body>
+    <h1>Welcome <span id="displayuser"></span>!</h1><br>
```

Comment:
You should handle the error case where there is no display user, in which case it just shows `Welcome !`. It looks awkward since there's a gramatically incorrect space between the word and the punctuation.